runtime/executor/src/main/java/org/apache/nemo/runtime/executor/datatransfer/InputWatermarkManager.java

```diff
@@ -27,7 +27,7 @@
  * --edge 2--&gt;  watermarkManager --(emitWatermark)--&gt; nextOperator
  * --edge 3--&gt;
  */
-public interface InputWatermarkManager {
+public interface InputWatermarkManager implements Serializable {
```

Comment:
please remove `implements Serializable`

Reply:
Thank you. I have modified all points that you have pointed out